src/main/java/uk/gov/hmcts/reform/civil/model/hearingvalues/ServiceHearingValuesModel.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.hmcts.reform.civil.model.hearingvalues;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Builder(toBuilder = true)
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class ServiceHearingValuesModel {
+
+    private String hmctsServiceID;
+    private String hmctsInternalCaseName;
+    private String publicCaseName;
+    private boolean caseAdditionalSecurityFlag;
+    private List<CaseCategoryModel> caseCategories;
+    private String caseDeepLink;
+    @JsonProperty("caserestrictedFlag")
```

Comment:
What is this JsonProperty used for?

Reply:
That was used to map the caseRestricedFlag field to the correct form for ExUI, as I think that expects "caserestrictedFlag"

Reply:
JSON annotation has now been removed

Reply:
we need this json annotation as the field that exui is expecting is called caserestrictedFlag however in java we have to camel case the field name. so i think this should be reverted @adam-thomas-HMCTS @HarryH96